### PR TITLE
Starter Kits: Take 2

### PR DESCRIPTION
**Pull request by jesseleite**
- [x] All the starter kit things!
    - [x] Polish cli messaging around starter kit license check. @jackmcdade?

Closes https://github.com/statamic/ideas/issues/501

Read the docs PR here for info on how to test: https://github.com/statamic/docs/pull/482

### CODE REVIEW

**Review by jasonvarga — CHANGES_REQUESTED** [review]
There are some merge conflicts that need to be resolved.

Also, let's target the `master` branch. This'll be 3.2.

File: src/StarterKits/LicenseManager.php [hunk 1]
```diff
@@ -73,7 +73,7 @@ public function expireLicense()
 
         Http::post(self::OUTPOST_ENDPOINT.'expire', [
```
Comment (duncanmcclean) [inline src/StarterKits/LicenseManager.php:74]: Just a quick thing to note - the `Http` stuff isn't included in Laravel 6 (which [Statamic currently supports](https://github.com/statamic/cms/blob/3.1/composer.json#L16)). It's available in Laravel 7 onwards.

Although, I guess it's very unlikely the SK license stuff will get run on a L6 install anyway so maybe this doesn't even matter?

Comment (jesseleite) [reply src/StarterKits/LicenseManager.php:74]: 👍  Yep, L7+ is required to install starter kits... https://github.com/statamic/cms/blob/starter-kits-2/src/Console/Commands/StarterKitInstall.php#L43-L45

Comment (duncanmcclean) [reply src/StarterKits/LicenseManager.php:74]: Gotcha! Sorry, I didn't read anything other than a little diff of changes 😅

Comment (jesseleite) [reply src/StarterKits/LicenseManager.php:74]: All good!

**Review by jasonvarga — APPROVED** [review]

**Pull request merged**